adapters/cf-worker/adapter.ts

```diff
@@ -23,10 +23,12 @@ export async function workerConvertRequest(
   for (const [key, value] of request.headers.entries()) {
     addHeader(headers, key, value);
   }
+
+  const url = new URL(request.url!);
```

Comment:
We were inconsistent in how we set up the URL for the normalized object for workers - this removes the hostname and protocol so the "internal" URL is just the path and later.